disnake/guild.py

```diff
@@ -185,9 +185,9 @@ class Guild(Hashable):
     description: Optional[:class:`str`]
         The guild's description.
     mfa_level: :class:`int`
-        Indicates the guild's two factor authorisation level. If this value is 0 then
-        the guild does not require 2FA for their administrative members. If the value is
-        1 then they do.
+        Indicates the guild's two-factor authentication level. If this value is 0 then
+        the guild does not require 2FA for their administrative members
+        to take moderation actions. If the value is 1 then it does.
```

Comment:
```suggestion
        to take moderation actions. If the value is 1 then the guild does.
```

Reply:
good call, adjusted it slightly more: 8571578754d66c892c4c093787aef16b94251146